packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -352,44 +360,79 @@ const PlanStep: React.FunctionComponent< PlanStepProps > = ( {
 						<div>
 							<FocusedLaunchSummaryItem
 								isLoading={ ! defaultFreePlan || ! defaultPaidPlan }
-								readOnly={ hasPaidDomain || selectedPaidDomain }
-								onClick={ () => defaultFreePlan && setPlan( defaultFreePlan ) }
-								isSelected={ ! ( hasPaidDomain || selectedPaidDomain ) && selectedPlan?.isFree }
+								onClick={ () => defaultPaidPlan && setPlan( defaultPaidPlan ) }
+								isSelected={ selectedPlan?.storeSlug === defaultPaidPlan?.storeSlug }
```

Comment:
The `isSelected` check could potentially return `true` if both `selectedPlan` and `defaultPaidPlan` are `undefined`. We should probably change this check to something like
```suggestion
								isSelected={ selectedPlan && selectedPlan.storeSlug === defaultPaidPlan?.storeSlug }
```